api/v1/stats/publications/PKPStatsPublicationHandler.php

```diff
@@ -35,6 +35,9 @@
 
 abstract class PKPStatsPublicationHandler extends APIHandler
 {
+    /** @var string The name of the section ids query param for this application */
+    public $sectionIdsQueryParam;
```

Comment:
hmmm... 
maybe I should use abstract public function getSectionIdsQueryParam() here, so that the child class do not forget to define it...